spec/json/BigCommerce_Orders_API.oas2.json

```diff
@@ -4683,6 +4683,12 @@
       "type": "boolean",
       "description": "If the message is [flagged](https://support.bigcommerce.com/s/article/Communicating-with-Customers#Messages)."
     },
+    "customer_id": {
+      "name": "customer_id",
+      "in": "query",
+      "type": "boolean",
```

Comment:
integer? Also the description doesn't look right

Reply:
@nikita-ahuja -- this still an issue?

Reply:
Oops. I fixed the mistype in [this commit](https://github.com/bigcommerce/dev-docs/pull/121/commits/43408fcded7c3d63cfa694f7d5d94dd66ab6b1f2) - the customer_id query parameter already exists and just needed to be referenced in the get /orders path.